content/blog/2020-06-08-june-20-dvc-heartbeat.md

```diff
@@ -0,0 +1,214 @@
+---
+title: June '20 Heartbeat
+date: 2020-06-08
+description: |
+  Catch up on new DVC releases, talks, and projects in our community. 
+  This month, learn about finishing touches on DVC 1.0, DVC in biomedical
+  research, recommended reading and upcoming MLOps talks.
+descriptionLong: |
+  Catch up on new DVC releases, talks, and projects in our community. 
+  This month, learn about finishing touches on DVC 1.0, DVC in biomedical
+  research, recommended reading and upcoming MLOps talks.
+picture: 2020-06-08/June_20_Heartbeat_small.png
+author: elle_obrien
+commentsUrl: https://discuss.dvc.org/t/june-20-heartbeat/404
+tags:
+  - Heartbeat
+  - Udemy Course
+  - Pipelines
+  - Plots
+  - MLOps
+---
+
+Welcome to the June Heartbeat, our monthly roundup of cool happenings,
+[good reads](#from-the-community) and
+[up-and-coming developments](#coming-up-soon) in the DVC community.
+
+## News
+
+In the beginning of May, we
+[pre-released DVC 1.0](https://dvc.org/blog/dvc-3-years-and-1-0-release). Ever
+since, we've been putting the final touches on 1.0- wrapping up features, fixing
+bugs 🐛, and responding to feedback from intrepid users
+[trying the pre-release](https://dvc.org/doc/install/pre-release). To recap,
+here are some of the big features coming:
+
+- **Plots powered by Vega-Lite** We're building
+  [functions for vizualizing metrics](<(https://dvc.org/doc/command-reference/plots#plots)>)
+  in your project, as well as comparing metrics across commits. We chose
+  [Vega-Lite plots](https://github.com/vega/vega-lite) because they're
+  high-level, compatible with ML projects written in any language, and beautiful
+  by default.
+
+- **Human readable and writeable pipelines.** We're reworking pipelines so you
+  can modify dependencies, outputs, metrics, plots, and entire stages easily:
+  via manual edits to a `.yaml` pipeline fines. This redesign will consolidate
+  pipline `.dvc` files into a single file (yay, simpler working directory).
+  We're also making helper scripts to ease the transition from classic DVC
+  pipelines to 1.0- pipeline enthusiasts, we have your back.
```

Comment:
Are we? @shcheklein

Reply:
@skshetry I'm not the author of this blog post and you should def have more context on this than I do :) so please advise and/or suggest edits and @andronovhopf can approve them.

Reply:
@shcheklein @andronovhopf Let's not promise for helper scripts, most likely it's not going to happen.

Reply:
It's backwards compatible anyway. And, new users don't have to use old-style dvcfiles at all.

I did try to create a script (and, it's not difficult  for a one-off), but maintaining it will be a pain considering how many times we change the file formats.

Reply:
If anyone wants that, I can direct to it, but I don't want to spend efforts maintaining it.

Reply:
Ah, got it. I knew you'd been working on a helper script but didn't realize it was going to be difficult to maintain. So I'll swap out for language like, "Don't worry, pipelines are backwards compatible."

Reply:
Have just opened up a PR to fix this